.gitignore

```diff
@@ -165,6 +165,9 @@ backend/detection_results.csv
 backend/firebase_helpers/bearer.py
 backend/firebase_helpers/firebase_creds.json
 
+#Poetry
+poetry.lock
```

Comment:
Commit the poetry.lock file: [https://python-poetry.org/docs/basic-usage/](https://python-poetry.org/docs/basic-usage/)

Reply:
Thanks for catching this. @codingwithsurya can you do that?

Reply:
I guess if we have `yarn.lock` in Version control, it shouldn't be suprising that `poetry.lock` should be there